### PR TITLE
Add \iff, \implies, \impliedby support

### LINKED ISSUE #190: Add support for \implies and \iff
These are tricky because they should have space around them (`\;` on either side). We should probably implement these as some sort of internal macro, which can maybe be a precursor to #37.

### PR DESCRIPTION
**Pull request by edemaine**
Fix #190 via @ronkok's code.

I thought about tests but couldn't think of anything that interesting.  A `expect(...).toParseLike` doesn't seem to check anything.  I could add a screenshot test, but it is also pretty "boring".

### CODE REVIEW

**Comment by ronkok** [comment]
That looks good. And kudos to @gagern, who made macros possible.

File: src/macros.js [hunk 1]
```diff
@@ -21,3 +21,10 @@ defineMacro("\\endgroup", "}");
 // \def\overset#1#2{\binrel@{#2}\binrel@@{\mathop{\kern\z@#2}\limits^{#1}}}
 defineMacro("\\overset", "\\mathop{#2}\\limits^{#1}");
 defineMacro("\\underset", "\\mathop{#2}\\limits_{#1}");
+
+// \def\iff{\DOTSB\;\Longleftrightarrow\;}
+// \def\implies{\DOTSB\;\Longrightarrow\;}
+// \def\impliedby{\DOTSB\;\Longleftarrow\;}
```
Comment (kevinbarabash) [inline src/macros.js]: Can you add a TODO above these to indicate that this is what we want the the implementation of these commands to eventually look like?

Comment (edemaine) [inline src/macros.js]: Added.

**Comment by edemaine** [comment]
@kevinbarabash Implemented both your suggestions (and rebased).  Thanks for the feedback!

I also noticed that, while @ronkok's code had a `\mathrel` surrounding the operators, the `amstex.sty` code did not, so I switched to the latter.

**Comment by ronkok** [comment]
I'm good with the change. The span still is coded as a `class="mrel"`.

**Comment by kevinbarabash** [comment]
@edemaine thanks for adding these new commands.

**Pull request merged**